src/routes/+layout.svelte

```diff
@@ -6,13 +6,19 @@
   import dayjs from "dayjs";
   /* Recommended for fraud detection */
   import "@stripe/stripe-js";
+  import { onMount } from "svelte";
+  import { setDefaultFlags } from "$lib/utils/featureFlag";
 
   export let data;
 
   $: (() => {
     const locale = languageTag();
     dayjs.locale(locale);
   })();
+
+  onMount(() => {
+    setDefaultFlags();
+  });
```

Comment:
Why do we always set the flags to false when loading? This causes all flags to become false on all page reloads.

Reply:
Oh yes that shouldn't be there, forgot to remove after testing some stuff